### ISSUE TITLE
Bug: can't format long checklist item lines

### ISSUE THREAD

**Issue by zolotokrylin** [rvben/rumdl#582]
### rumdl version

0.1.80

### Installation method

cargo install

### Bug description

Looks like there is a loop; the formatter is trying to align the next line with the first letter of the list item, while the MD013 wants to indent to make the line shorter "Line length 92 exceeds 80 characters".

<img width="749" height="117" alt="Image" src="https://github.com/user-attachments/assets/b25e83f3-63d2-4d25-a96b-bb8e3bbda23e" />

Basically, it makes it impossible to fix warnings in checklist items when they are longer than the allowed line. There is no rule to set specifically for checklist items.

As a workaround, I can use an inline "ignore" directive comments, but it is not very user-friendly to do it for each checklist item.

### Steps to reproduce

1. create a new file with the content (see bellow) 
2. use my config (see bellow)
3. run `rumdl check --fix` 

```md
# Content

- [ ] @holdex/hr-payroll-operations: post additional costs in own payout issue
  and link here
```

### Expected behavior

Allow checklist items to be as long as the user wants.

### Actual behavior

Described above

### Configuration

```toml
# rumdl configuration file

# Global configuration options
[global]
unfixable = ["MD013"]
disable = []
# List of rules to disable (uncomment and modify as needed)
# disable = ["MD013", "MD033"]

# List of rules to enable exclusively (if provided, only these rules will run)
# enable = ["MD001", "MD003", "MD004"]

# List of file/directory patterns to include for linting (if provided, only these will be linted)
# include = [
#    "docs/*.md",
#    "src/**/*.md",
#    "README.md"
# ]

# List of file/directory patterns to exclude from linting
exclude = [
    # Common directories to exclude
    ".git",
    # ".github",
    "node_modules",
    "vendor",
    "dist",
    "build",

    # Specific files or patterns
    # "CHANGELOG.md",
    # "LICENSE.md",
]

# Respect .gitignore files when scanning directories (default: true)
respect-gitignore = true

# Markdown flavor/dialect (uncomment to enable)
# Options: mkdocs, gfm, commonmark
# flavor = "mkdocs"

# Rule-specific configurations (uncomment and modify as needed)

# [MD003]
# style = "atx"  # Heading style (atx, atx_closed, setext)

# [MD004]
# style = "asterisk"  # Unordered list style (asterisk, plus, dash, consistent)

# [MD007]
# indent = 4  # Unordered list indentation

[MD010]
spaces-per-tab = 2

[MD013]
line-length = 80  # Line length
code-blocks = false  # Exclude code blocks from line length check
tables = false  # Exclude tables from line length check
headings = true  # Include headings in line length check
reflow = true  # Enable automatic line wrapping (required for --fix)
reflow-mode = "normalize"  # Reflow mode: "default", "normalize", or "sentence-per-line" (default: "default")

# [MD044]
# names = ["rumdl", "Markdown", "GitHub"]  # Proper names that should be capitalized correctly
# code-blocks = false  # Check code blocks for proper names (default: false, skips code blocks)

[MD029]
# Style options:
# - "one" or "one-one": All items numbered "1." (easiest maintenance)
# - "ordered": Sequential numbering (1, 2, 3...)
# - "ordered0": Zero-based sequential (0, 1, 2...)
# - "one-or-ordered": Auto-detect per list (either all-ones OR sequential)
# - "consistent": Document-wide consistency - uses most common style across all lists
style = "one"  # Default - matches markdownlint behavior


[MD033]
allowed-elements = ["br", "hr", "details", "summary"]

[MD046]
style = "consistent"  # Options: "consistent", "fenced", "indented"
```

### Operating system

macOS

**Comment by rvben** [comment]
Fixed in [v0.1.81](https://github.com/rvben/rumdl/releases/tag/v0.1.81). Thanks for the report, @zolotokrylin!
